examples/error_monster/pipeline.py

```diff
@@ -0,0 +1,80 @@
+from dagster import (
+    DependencyDefinition,
+    Field,
+    InputDefinition,
+    Int,
+    Bool,
+    String,
+    Dict,
+    OutputDefinition,
+    PipelineDefinition,
+    SolidInstance,
+    RunConfig,
+    execute_pipeline,
+    solid,
+)
+
+
+@solid(
+    name='start',
+    outputs=[OutputDefinition(Int)],
+    config_field=Field(Dict(fields={'throw_in_solid': Field(Bool)})),
+)
+def emit_num(context):
+    if context.solid_config['throw_in_solid']:
+        raise Exception('throwing from in the solid')
+
+    return 69
```

Comment:
let's change this number before checkin so that it is not intepreted as a crude joke

Reply:
🙃